### DISCUSSION TITLE
Project field of type select should have an additional type

### DISCUSSION THREAD

**Discussion by jlucaspains** [community/community#31894] [Projects and Issues]
Single selection field types are automatically of text type. This works for many scenarios, but I want to create a Effort field for a Scrum like functionality where only specific values can be used (1, 3, 5, 8, etc). Because this is stored as a text, I can't use a sum over the values selected in Insights. This would be very useful to identify the total effort in a project or sprint.

**Comment by dwainm** [comment]
It would be fantastic if the type could also be detected based on the values. If you have a select type with only numeric values, it should become available as a summable field.
